docs/en/sql-reference/dictionaries/external-dictionaries/external-dicts-dict-sources.md

```diff
@@ -523,6 +525,8 @@ Setting fields:
 
 -   `invalidate_query` – Query for checking the dictionary status. Optional parameter. Read more in the section [Updating dictionaries](../../../sql-reference/dictionaries/external-dictionaries/external-dicts-dict-lifetime.md).
 
+-   `fail_on_connection_loss` – The configuration parameter that controls unexpected connection loss during query execution. By default, false. ClickHouse connects to MySQL servers using secure protocols ([SSL](../../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-openssl)/[TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security)). Occasionally errors might occur in the transport communication between client and server, in which case the server would abort the client connection. Note that retrying leads to increased response times. Retries can be avoided by setting the parameter to true.
```

Comment:
It is better to make it concise, seems like too much text and no actual info. The main point: If `fail_on_connection_loss` is `true`, then there will be an exception about connection loss straight away if there was no connection when query was executed. If  `fail_on_connection_loss` is `false`, then there will be attempts to retry queries `n` times. where `n` is not configurable now and default (`n = 3`) is always used.

Reply:
> It is better to make it concise, seems like too much text and no actual info. The main point: If `fail_on_connection_loss` is `true`, then there will be an exception about connection loss straight away if there was no connection when query was executed. If `fail_on_connection_loss` is `false`, then there will be attempts to retry queries `n` times. where `n` is not configurable now and default (`n = 3`) is always used.

Ok, thank you. I will fix it.